src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -47,6 +53,21 @@
     <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\AES\AesFactory.cs">
       <Link>CommonTest\AlgorithmImplementations\AES\AesFactory.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\DSA\DSAFactory.cs" Condition="'$(TargetGroup)'=='netstandard1.7'">
```

Comment:
I think that when it's new files that are being added, in theory it shouldn't be a problem as long as it builds. This is because customers will be building against the old (netstandard1.6) version of the reference assembly, so they wouldn't se the new types/members even though they exist at runtime. That said, I think that if the implementation can only be used at netstandard1.7, then probably we should just add that file when we are building for ns1.7. @ericstj thoughts?